R/factset.protobuf.stachextensions/R/V2ColumnOrganizedStachUtilities.R

```diff
@@ -0,0 +1,80 @@
+#' @docType class
+#' @title V2ColumnOrganizedStachUtilities
+#' @description Provide helper functions for column organized stach
+#'
+#' @importFrom R6 R6Class
+#' @export
+V2ColumnOrganizedStachUtilities <- R6::R6Class(
```

Comment:
Can you please update the library name something like "V2ColumnOrganizedStachDecompress" ? because utilities is not completely opting for this scenario

Reply:
@vamshimupparaju Let's go with the current name itself. It can help us in the future to add more helper methods in the same class. Is that fine?